.eslintcache

```diff
@@ -1 +1 @@
ashOfConfig":"8"},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},"1wo60u1",{"filePath":"17","messages":"18","errorCount":0,"warningCount":2,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"25","messages":"26","errorCount":1,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"source":"27","usedDeprecatedRules":"28"},"/Users/a/Desktop/react-tinder-cards/src/index.js",[],["29","30"],"/Users/a/Desktop/react-tinder-cards/src/App.js",["31","32"],"/Users/a/Desktop/react-tinder-cards/src/reportWebVitals.js",[],"/Users/a/Desktop/react-tinder-cards/src/components/profile-card.js",[],"/Users/a/Desktop/react-tinder-cards/src/network/index.js",[],"/Users/a/Desktop/react-tinder-cards/src/utils/index.js",["33"],"import Logger from 'js-logger';\n\nLogger.useDefaults();\n\nexport function getLogger(name){\n    \n    return Logger.get(name);\n}",["34","35"],{"ruleId":"36","replacedBy":"37"},{"ruleId":"38","replacedBy":"39"},{"ruleId":"40","severity":1,"message":"41","line":54,"column":6,"nodeType":"42","endLine":54,"endColumn":29,"suggestions":"43"},{"ruleId":"40","severity":1,"message":"44","line":59,"column":6,"nodeType":"42","endLine":59,"endColumn":17,"suggestions":"45"},{"ruleId":"46","severity":2,"message":"47","line":3,"column":1,"nodeType":"48","endLine":3,"endColumn":19},{"ruleId":"36","replacedBy":"49"},{"ruleId":"38","replacedBy":"50"},"no-native-reassign",["51"],"no-negated-in-lhs",["52"],"react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'profiles'. Either include it or remove the dependency array. You can also do a functional update 'setProfiles(p => ...)' if you only need 'profiles' in the 'setProfiles' call.","ArrayExpression",["53"],"React Hook useEffect has missing dependencies: 'currentProfileIndex' and 'profiles.length'. Either include them or remove the dependency array.",["54"],"react-hooks/rules-of-hooks","React Hook \"Logger.useDefaults\" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function.","MemberExpression",["51"],["52"],"no-global-assign","no-unsafe-negation",{"desc":"55","fix":"56"},{"desc":"57","fix":"58"},"Update the dependencies array to be: [profiles, shouldPrefetchProfile]",{"range":"59","text":"60"},"Update the dependencies array to be: [currentProfileIndex, isLoading, profiles.length]",{"range":"61","text":"62"},[1756,1779],"[profiles, shouldPrefetchProfile]",[1922,1933],"[currentProfileIndex, isLoading, profiles.length]"]
```

Comment:
this should be gitignored. btw do you use a mac or PC?

Reply:
I've tried to add this in `.gitignore` but it doesn't seem to work, don't know why.  I'll try again 😄 
I use Mac 💻